mmd_tools/operators/fileio.py

```diff
@@ -260,6 +260,9 @@ def execute(self, context):
             bpy.ops.pose.transforms_clear()
             bpy.ops.object.mode_set(mode='OBJECT')
             root.mmd_root.show_armature = prev_show
+        # Add extension if needed
+        if not self.filepath.endswith(self.filename_ext):
+            self.filepath += self.filename_ext 
```

Comment:
I think the `ExportHelper` will do this trick, so this changes might not be necessary!